server/src/main/resources/rules/rules.js

```diff
@@ -3818,10 +3837,13 @@ var PoolType = {
 }
 
 /**
- * Namespace for determining ability to override a specific content set
- *  value
+ * Namespace for determining ability to override a specific content set value
  *
  * The list of disallowed value names are listed in this method
+ *
+ * DEPRECATED:
+ *  This functionality has been moved to ContentOverrideValidator as of Candlepin 2.5.5. This method
+ *  will no longer be called by Candlepin and is considered deprecated.
```

Comment:
The addition of these comment has nothing to do with the actual changes for the issue at hand. It is simply done to keep the rules in hosted (candlepin-2.3-HOTFIX) in sync with master/satellite branches.